plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTest.java

```diff
@@ -219,8 +295,8 @@ public void remove_success() {
   }
 
   @Test
-  public void remove_error() {
-    // Mock the side effects of removePackFailureTaskMock object, call onFailureListener the instant
+  public void removePack_error() {
+    // Mock the side effects of Task<Void> object, call onFailureListener the instant
```

Comment:
I changed this commented since it will be clearer for first time readers (especially without IDE)